File: go_http2/app/src/rabbitmq/queues.go
package rabbitmq

const (
	ExchangeName      = "go-http2"
	QueueGolang       = "go-http2"
	QueueSystemGolang = "system-go-http2"
)

// GetQueueName возвращает название очереди в зависимости от типа события
func GetQueueName(isSystem bool) string {
	if isSystem {
		return QueueSystemGolang
	}
	return QueueGolang
}

File: rust_axum/app/src/rabbitmq/queues.rs
pub const EXCHANGE_NAME: &str = "rust-axum";
pub const QUEUE_AXUM: &str = "rust-axum";
pub const QUEUE_SYSTEM_AXUM: &str = "system-rust-axum";

pub fn get_queue_name(is_system: bool) -> &'static str {
    if is_system {
        QUEUE_SYSTEM_AXUM
    } else {
        QUEUE_AXUM
    }
}
